doc/tutorials/rtvm_workload_design_guideline.rst

```diff
@@ -0,0 +1,42 @@
+.. _rtvm_workload_guideline:
+
+Guidelines to the workload design in an RTVM
+############################################
+
+For a RTOS designer/developer, if he/she want to run the RTOS based on ARCN RTVM,
```

Comment:
It's best to avoid using ``he/she``, so instead, write this as:
```
An RTOS developer must be aware of the differences between
running applications on a native platform and a real time virtual
machine (RTVM), especially issues impacting real time
performance.  For example, a real time thread should
avoid using VM-Exit.  If VM-Exit is used, the developer must
account for an additional margin of CPU cycles for
the added run time cost.

This document provides some workload design guidelines when
using an RTVM within the ACRN hypervisor.
```

Reply:
@dbkinder , thanks for the comments, please review the new commit